setup.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+cd
```

Comment:
This one's my bad, can you pls move this to right above "checking if bbctl is installed" on line 109?